experimental/web_dashboard/README.md

```diff
@@ -30,3 +30,107 @@ Skia / CanvasKit mode:
 flutter run -d chrome --release --dart-define=FLUTTER_WEB_USE_SKIA=true
 ```
 
+## Running JSON code generator
+
+```
+flutter pub run grinder generate
+```
+
+## Add Firebase
+
+### Step 1: Create Firebase project and install packages
+
+Follow the steps at [firebase.google.com/docs/flutter/setup][flutter-setup].
+This app uses these Firebase packages:
+
+```yaml
+# Add these packages to your pubspec.yaml under "dependencies"
+  firebase_core: ^0.4.3
+  cloud_firestore: ^0.13.0
+  firebase_auth: ^0.15.0
+  google_sign_in: ^4.0.0
+```
+
+### Step 2: Enable Google Sign In for your project
+
+In the Firebase console, go to "Authentication" and enable Google sign in. Click
+on "Web SDK Configuration" and copy down your Web client ID.
+
+### Step 3: Add firebase scripts to `index.html`
+
+Add these scripts to the `<head>` tag:
```

Comment:
These files are already linked into index.html in this project, right?

I think I may be misinterpreting the purpose of these instructions. Are these meant as general guidelines for getting Firebase up and running in a Flutter app built for web, or for this app specifically? If it's the former, I'm not sure that content fits this location. If it's the latter, I'd recommend cutting anything already done in the project.

Reply:
Decided go with option 2. and made the instructions specific for this sample 7e7df2bf91da613e40d67c9502d77be2d8d7fd03